src/gameLogic/execution/ActionExecutor_Spy.cpp

```diff
@@ -2,16 +2,108 @@
 // Created by jonas on 28.04.20.
 //
 #include "ActionExecutor.hpp"
+#include <util/GameLogicUtils.hpp>
 
 namespace spy::gameplay {
+
+    static std::shared_ptr<const BaseOperation>
+    handleSpyOnPerson(State &s, const SpyAction &op, const MatchConfig &config,
```

Comment:
forward declaration bitte in header

Reply:
(oder wenn du die hier lassen willst, halt nicht forward declaren sondern direkt hier hinschreiben)